src/axom/core/CMakeLists.txt

```diff
@@ -54,21 +59,28 @@ set(core_sources
     utilities/Utilities.cpp
     utilities/About.cpp
 
+    utilities/nvtx/interface.cpp
+    utilities/nvtx/Range.cpp
+
     numerics/polynomial_solvers.cpp
     Types.cpp
     )
 
 #------------------------------------------------------------------------------
-# Make/Install the library
+# Set library dependencies
 #------------------------------------------------------------------------------
 
 set( core_depends )
 blt_list_append( TO core_depends ELEMENTS umpire IF ${UMPIRE_FOUND} )
 blt_list_append( TO core_depends ELEMENTS RAJA IF ${RAJA_FOUND} )
-blt_list_append( TO core_depends ELEMENTS cuda IF ${ENABLE_CUDA} )
+blt_list_append( TO core_depends ELEMENTS cuda nvToolsExt IF ${ENABLE_CUDA} )
```

Comment:
Is ``nvToolsExt`` always available when ``cuda`` is?

Do we need to update the documentation about axom's dependencies?

Reply:
Should it be explicitly defined, e.g. via ``blt_register_library`` ?

Reply:
Yes -- `nvToolsExt` is a part of the NVIDIA SDK. There is no need to "register" it as a library

Reply:
@gzagaris -- it looks like there was an issue with `nvToolsExt` on our `xl@nvcc` host-config for `blueos_p9`.

Can you please look into it?